packages/block-library/src/cover/edit/index.js

```diff
@@ -514,6 +514,8 @@ function CoverEdit( {
 								value={ overlayColor.color }
 								onChange={ onSetOverlayColor }
 								clearable={ false }
+								asButtons
+								aria-label={ __( 'Background color' ) }
```

Comment:
Nit: In the cover block, it is not called background color but overlay.
Even if there is no media selected.